mep/people/models.py

```diff
@@ -297,6 +297,11 @@ def has_account(self):
         return self.account_set.exists()
     has_account.boolean = True
 
+    def is_creator(self):
```

Comment:
Are we using this anywhere?

Reply:
I created it so that I could use it in the test for the filter, as a sanity check similar to `has_account()`. Figured it might eventually see use somewhere else.